packages/react/src/patterns/sub-patterns/Card/Card.js

```diff
@@ -27,10 +27,12 @@ const { prefix } = settings;
  */
 const Card = ({
   title,
+  eyebrow,
   copy,
   href,
   cta,
   image,
+  inverse,
   className,
   type,
   icon: Icon,
```

Comment:
We'll probably need to updated this to the new SVG data model we were talking about.

```js
icon: {
  src: Icon,
  ...iconProps
}
```

Reply:
let's put this in another PR